x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/logstash_instructions/index.tsx

```diff
@@ -34,21 +34,21 @@ export const LogstashInstructions = () => {
       title={
         <FormattedMessage
           id="xpack.fleet.settings.logstashInstructions.calloutTitle"
-          defaultMessage="Configure Logstash for Elastic Agent"
+          defaultMessage="You must configure Logstash for Elastic Agent"
         />
       }
     >
       <>
         <FormattedMessage
           id="xpack.fleet.settings.logstashInstructions.description"
-          defaultMessage="You must add a pipeline for Elastic Agent to Logstash. For more information, visit our
+          defaultMessage="You must add an Elastic Agent pipeline configuration to Logstash.
```

Comment:
We might want to think about this one a little more.

- Both the title and sentence begin with "You must"
- There's a Learn more link, but the instructions are right below it.

Maybe just this text is needed as the title? And no intro sentenced needed?

> Add an Elastic Agent pipeline configuration to Logstash

Reply:
@gchaps The "Learn more" link points to reference content that describes Logstash config options (it doesn't duplicate the steps shown in the UI). Users need to modify the code in the example. In fact, they'll want to add some other stages to do actual processing. So I think it's worth sending them to the reference docs.

I'll play around with the text. I think we can make the intro more useful.